content/posts/02_staff_expectations/index.md

```diff
@@ -0,0 +1,127 @@
+---
+title: "What is expected from staff engineers"
+date: 2022-12-18
+tags: [
+    "career", "growth"
+]
+---
+
+It's relatively easy to explain expectations for junior, mid, and even senior engineers. But it starts to get much harder at staff+ 
+positions. Career progression broadens the scope of work and responsibilities. The environment and problems get more ambiguous. It's demanding
+to define a clear and measurable expectations, because staff roles also differ between companies and
+their actual needs. They can also vary between different [staff engineer archetypes](https://staffeng.com/guides/staff-archetypes).
+
+In the article, we will focus mostly on the **Tech Lead** archetype, which is the most popular in the industry. We will present some 
+expectations for staff engineers based on our experience. The list is incomplete and there are many very important skills and 
+traits a strong staff engineer should have. We wanted to describe those that are fundamental regardless of the specific situation.
+
+# So what are those expectations?
+
+## Get things done
+![](./done.jpg)
+
+This might as well be applied to the earlier positions, however, it assumes an even greater significance for staff engineers. Staff engineers 
+shoulder the responsibility of accomplishing their tasks proficiently while also facilitating the success of their peers. They must skillfully 
+aid other engineers and teams in managing and prioritizing intricate projects, serving as the crucial "glue" that cements the smooth functioning
+and timely delivery of engineering projects. Moreover, they should attentively identify and eliminate any hindrances that impede the development
+experience while simultaneously enhancing productivity.
+
+Some of the examples of getting stuff done:
+* Help the team with project management and prioritization.
```

Comment:
`Help teams with their project management and prioritization` - I guess we're not referring to any specific team here.